Doc/cookbook/Restriction/Restriction.md

```diff
@@ -127,16 +127,11 @@ KpnI
 
 So what can we do with these restriction enzymes? To see that we will need a
 DNA sequence. Restriction enzymes support both `Bio.Seq.MutableSeq`and
-`Bio.Seq.Seq` objects. You can use any DNA alphabet which complies with the
-IUPAC alphabet.
+`Bio.Seq.Seq` objects. You sequence must comply with the IUPAC alphabet.
```

Comment:
Your sequence

Reply:
Also users may not know what the IUPAC alphabet is.

Reply:
Good points, that can be improved.